packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionStart.test.ts

```diff
@@ -5,15 +5,15 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
```

Comment:
I'll rename this to indicate it's an integration test for the container start flow.